crates/routes/src/nodeinfo.rs

```diff
@@ -9,8 +9,16 @@ use url::Url;
 
 pub fn config(cfg: &mut web::ServiceConfig) {
   cfg
-    .route("/nodeinfo/2.0.json", web::get().to(node_info))
-    .route("/.well-known/nodeinfo", web::get().to(node_info_well_known));
+    .route(
+      "/nodeinfo/2.0.json",
+      web::get().to(node_info).wrap(lemmy_utils::cache_1hour()),
+    )
+    .route(
+      "/.well-known/nodeinfo",
+      web::get()
+        .to(node_info_well_known)
+        .wrap(lemmy_utils::cache_3days()),
```

Comment:
So all the data in NodeInfo could be 3 days old?

Reply:
This is essentially just a redirect to the `/nodeinfo/2.0.json` endpoint with completely static data. No harm in caching it long time.